src/polyfill-globals.js

```diff
@@ -15,38 +15,32 @@
 
 import XRPresentationContext from './api/XRPresentationContext';
 import {
-  POLYFILLED_COMPATIBLE_XR_DEVICE,
-  COMPATIBLE_XR_DEVICE,
+  POLYFILLED_XR_COMPATIBLE,
+  XR_COMPATIBLE,
 } from './constants';
 
 const contextTypes = ['webgl', 'experimental-webgl'];
 
 /**
  * Takes the WebGLRenderingContext constructor
- * and creates a `setCompatibleXRDevice` function if it does not exist.
+ * and creates a `makeXRCompatible` function if it does not exist.
  * Returns a boolean indicating whether or not the function
  * was polyfilled.
  *
  * @param {WebGLRenderingContext}
  * @return {boolean}
  */
-export const polyfillSetCompatibleXRDevice = Context => {
-  if (typeof Context.prototype.setCompatibleXRDevice === 'function') {
+export const polyfillMakeXRCompatible = Context => {
```

Comment:
If we have challenges with this area, I'd be ok with removing it; idea is to be able to test with the polyfill and it can catch early errors like not making the context compatible even if you don't have access to a working WebXR browser (although, less of an issue now I think thanks to GVR+Chrome?) -- or can handle in a follow up